assets/styles/common/elements.scss

```diff
@@ -1,6 +1,7 @@
 @import 'bootstrap';
 
 .structured-content,
+.activity-editor,
```

Comment:
There was a bug where none of this formatting was being applied in the authoring environment within activities, adding activity-editor in here fixed that.

The formatting does appear in the lesson itself, so presumably we'd want the author to see it too.